pkgs/applications/audio/termusic/default.nix

```diff
@@ -47,6 +48,7 @@ rustPlatform.buildRustPackage rec {
     darwin.apple_sdk.frameworks.IOKit
     darwin.apple_sdk.frameworks.MediaPlayer
     darwin.apple_sdk.frameworks.Security
+    SystemConfiguration
```

Comment:
```suggestion
    darwin.apple_sdk.frameworks.SystemConfiguration
```

Reply:
I fixed it myself.

Reply:
Thanks, I was unsure if this prefix was as needed, but thought the other package build well without it.. sorry for the fuzz.

Reply:
No problem :)